projects/1.16/assets/create/create/lang/zh_cn.json

```
@@ -12,7 +12,7 @@
   "block.create.andesite_bricks_stairs": "安山岩砖楼梯",
   "block.create.andesite_bricks_wall": "安山岩砖墙",
   "block.create.andesite_casing": "安山机壳",
-  "block.create.andesite_cobblestone": "安山岩圆石",
+  "block.create.andesite_cobblestone": "安山圆石",

```

| **审查级别** |
|:-:|
| ℹ️ 信息 / 不阻止合并  |

**原文: Andesite Cobblestone**

```suggestion
  "block.create.andesite_cobblestone": "安山圆石",
```
Test Content